systers_portal/templates/meetup/add_meetup.html

```diff
@@ -39,9 +39,9 @@
   })
 
   $('.timepicker').pickatime({
-    format: 'HH:i',
-    formatSubmit: 'HH:i',
+    format: 'H:i',
+    formatLabel: 'h:ia',
```

Comment:
The accepted formats for TimeField are https://docs.djangoproject.com/en/1.8/ref/forms/fields/#timefield
Why would you switch to 12 hour format and day time period?